src/components/_proxied-dot-io/common/io-home-inline-callout/style.module.css

```diff
@@ -0,0 +1,53 @@
+.inlineCallout {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  padding: 32px;
+  gap: 16px 32px;
+  background-color: var(--brand-secondary);
```

Comment:
⛏️ Hmm is there a lighter variant of `--brand-secondary` that could be used? Just noticing the background in preview is darker than screenshots in the Asana thread:

![chrome_dZVx2NReks](https://user-images.githubusercontent.com/3746694/169388516-1d6acdb9-ed2f-4fe9-940d-1f219bd73c58.png)

Or would changing this adversely affect other product pages?

Reply:
Yeah, goal is to use `--brand-secondary` to match the other elements on the page

![CleanShot 2022-05-19 at 15 41 36@2x](https://user-images.githubusercontent.com/825855/169389328-9ba98e38-cca8-4516-8084-e67c6a1a947e.png)

<img width="1279" alt="CleanShot 2022-05-19 at 15 42 02@2x" src="https://user-images.githubusercontent.com/825855/169389344-d2397764-caeb-42d1-8ced-d699ac065c2b.png">

Reply:
Ah yes, good call!